src/ExpressionEvaluator/CSharp/Test/ResultProvider/ValueFormattingTests.cs

```diff
@@ -214,8 +214,8 @@ public void String()
             Assert.Equal("\"\\uffff\"", FormatValue(s, useHexadecimal: true));
 
             string multiByte = "\ud83c\udfc8";
-            Assert.Equal(string.Format(format, "🏈"), FormatValue(multiByte));
-            Assert.Equal(string.Format(format, "🏈"), FormatValue(multiByte, useHexadecimal: true));
+            Assert.Equal(string.Format(format, "\\ud83c\\udfc8"), FormatValue(multiByte));
+            Assert.Equal(string.Format(format, "\\ud83c\\udfc8"), FormatValue(multiByte, useHexadecimal: true));
```

Comment:
Yes, the documentation is somewhat lacking. The implementation, however, is here: https://source.dot.net/#System.Private.CoreLib/shared/System/Char.cs,517b834a1717ca04.

It ends up converting the character at `index` to `Utf32` (https://source.dot.net/#System.Private.CoreLib/src/System/Globalization/CharUnicodeInfo.cs,553115e31e8da9c0) and then passes the resulting `int` to the internal lookup table.

The regular `char.GetUnicodeCategory(char)` function does effectively the same but cannot handle surrogate characters due to the limitations of `char` (which can only represent a single utf-16 codepoint).